ProcessesApi.Tests/V1/HttpHeadersExtensionsTests.cs

```diff
@@ -9,7 +9,7 @@
 using System.Linq;
 using System.Threading.Tasks;
 
-namespace BaseApi.Tests.V1
+namespace ProcessesApi.Tests.V1
```

Comment:
This file can be deleted and you can use to the Middleware nuget package instead